go-app-ussd_nurse_rapidpro.js

```diff
@@ -36,24 +37,54 @@ go.app = function() {
 
             /*
                 find identity & check subscription using CompanionApp
-
+                use temporary fixture to differentiate non_subscribers and subscribers
+                this will be replaced with method that checks subscriptions
                 will go to state_registered if found,
                 else state_not_registered
             */
-            return self.states.create('state_not_registered');
-
+            if(msisdn === "+27820001003"){ 
+                return self.states.create('state_registered_1');
+            } else {
+                return self.states.create('state_not_registered');
+            }
         });
 
         self.states.add('state_not_registered', function(name){
             return new FreeText(name, {
                 question: ("Welcome to NurseConnect, where you can stay up to date with "+
                 "maternal & child health. Reply '1' to start."),
-                next: 'state_nurse_connect_options'
+                next: 'state_not_registered_menu'
+            });
+        });
+
+        self.states.add('state_registered_1', function(name) {
+            return new MenuState(name, {
+                question: ("Welcome back to NurseConnect. Do you want to:"),
+                choices: [
+                    new Choice('state_friend_registration', ('Help a friend sign up')),
```

Comment:
people can sign their friends up for nurse connect? Shuu

Reply:
not friend, but a fellow nurse "friend". The state option is called `Help a friend register`. Do you find it better that the state be named differently?

Reply:
colleague maybe?
Friend sounds like ichommie zakho

Reply:
I'll rename it to "other_registration". See comment bellow